extensions/wikia/WikiaMaps/controllers/WikiaMapsPoiCategoryController.class.php

```diff
@@ -29,6 +29,14 @@ public function getParentPoiCategories() {
 	 * @throws BadRequestApiException
 	 */
 	public function editPoiCategories() {
+		// Temporary change required for ad purpose - https://wikia-inc.atlassian.net/browse/DAT-4051.
+		// We need to limit contribution options on protected maps related to the ad campaign only to stuff
+		// users.
+		// TODO: remove this as a part of https://wikia-inc.atlassian.net/browse/DAT-4055
+		if ( $this->shouldDisableProtectedMapEdit( $this->request->getInt( 'mapId' ) ) ) {
```

Comment:
I think we don't need the word 'Protected' here - it's up to the method to decide how the shouldDisabledMapEdit is handled